kuma/vagrant-wsgi.py

```diff
@@ -0,0 +1,14 @@
+import os
```

Comment:
The goal (once we're on AWS) is to not have a `wsgi/kuma.wsgi` anymore as that's an artifact of the way we currently run MDN via Apache w/ mod_wsgi. On AWS we'll be using Nginx and uwsgi.